acceptance/tests/language/sensitive_data_type.rb

```diff
@@ -75,13 +84,20 @@
           run_assertions(assertion_code, result)
         end
 
-        if agent['platform'] =~ /fedora|centos|el|redhat|scientific|ubuntu|debian|cumulus/
```

Comment:
it probably needs an else logger.warn "we're not testing logs on #{platform}" or something, but i wouldn't hold up this PR on that